cbmc/proofs/IotMqtt_UnsubscribeAsync/Makefile

```diff
@@ -0,0 +1,117 @@
+ENTRY=IotMqtt_UnsubscribeAsync
+
+#CBMCFLAGS += --verbosity 10
+
+################################################################
+# Proof assumptions
+
+# Bound the number of subscriptions in a list (BOUND = MAX-1)
+SUBSCRIPTION_COUNT_MAX=2
+DEF += -DSUBSCRIPTION_COUNT_MAX=$(SUBSCRIPTION_COUNT_MAX)
+
+# Bound the number of operations in a list (BOUND = MAX-1)
+OPERATION_COUNT_MAX=2
+DEF += -DOPERATION_COUNT_MAX=$(OPERATION_COUNT_MAX)
+
+# Bound the length of subscription topics (BOUND = MAX-1)
+TOPIC_LENGTH_MAX=8
+DEF += -DTOPIC_LENGTH_MAX=$(TOPIC_LENGTH_MAX)
+
+# A constant that should be 2*SUBSCRIPTION_COUNT_MAX-1
+SUBSCRIPTION_LIST_MAX=3
+DEF += -DSUBSCRIPTION_LIST_MAX=$(SUBSCRIPTION_LIST_MAX)
+
+################################################################
+# MQTT configuration constants modified for testing
+
+# this should be smaller than topic_length_max for coverage
+# _validateSubscription coverage at line 385
+AWS_IOT_MQTT_SERVER_MAX_TOPIC_LENGTH=6
+DEF += -DAWS_IOT_MQTT_SERVER_MAX_TOPIC_LENGTH=$(AWS_IOT_MQTT_SERVER_MAX_TOPIC_LENGTH)
+
+# this should be smaller than topic_length_max for coverage
+# _IotMqtt_SubscriptionPacketSize coverage at line 702
+MQTT_MAX_REMAINING_LENGTH=6
+DEF += -DMQTT_MAX_REMAINING_LENGTH=$(MQTT_MAX_REMAINING_LENGTH)
+
+################################################################
+# Function omitted from the proof
+
+ABSTRACTIONS += --remove-function-body IotLog_Generic
+
+# Functions removed for coverage
+ABSTRACTIONS += --remove-function-body _IotMqtt_AddSubscriptions
+ABSTRACTIONS += --remove-function-body _IotMqtt_SerializeSubscribe
+ABSTRACTIONS += --remove-function-body _IotMqtt_SerializeSubscribeCommon
+ABSTRACTIONS += --remove-function-body _matchEndWildcards
+ABSTRACTIONS += --remove-function-body _matchWildcards
+ABSTRACTIONS += --remove-function-body _mqttOperation_match
+ABSTRACTIONS += --remove-function-body _mqttSubscription_setUnsubscribe
+ABSTRACTIONS += --remove-function-body _packetMatch
+ABSTRACTIONS += --remove-function-body _topicFilterMatch
+ABSTRACTIONS += --remove-function-body _validateQos
+ABSTRACTIONS += --remove-function-body IotNetworkInterfaceClose
+ABSTRACTIONS += --remove-function-body IotNetworkInterfaceReceive
```

Comment:
Does any of these functions appear as "Expected functions" on the report? If yes, please highlight them; otherwise, ignore this comment.

Reply:
No, none of these appear as missing functions on the report.